.gitignore

```diff
@@ -1,3 +1,7 @@
+# SPDX-FileCopyrightText: 2012 meta-raspberrypi contributors
+#
+# SPDX-License-Identifier: CC0-1.0
```

Comment:
Isn't this changing the license of the non-code-meta-data parts of this repository? This file used to be MIT by the "umbrella" license specification we had until now - now we switch it to CC0. Is this fine because of compatibility?

Also, why using this license for this file?